Volume/Supporting/Main.swift

```diff
@@ -12,9 +12,11 @@ import SwiftUI
 
 @main
 struct Main: App {
+    @UIApplicationDelegateAdaptor(AppDelegate.self) private var appDelegate
+    
     init() {
         configureFirebase()
-
+        registerForPushNotifications()
         let grayColor = UIColor(Color.volume.navigationBarGray)
         UINavigationBar.appearance().backgroundColor = grayColor
```

Comment:
Why do you define grayColor above?  Is it to prevent line 21 from getting too long?